config/build.ts

```diff
@@ -430,14 +436,16 @@ function checkSnippetsForUniqueIDs(processedSnippets) {
     }
 }
 
-async function generatePlaylists(dir, processedSnippets: Dictionary<SnippetProcessedData>) {
+async function generatePlaylists(dir, processedSnippets: Dictionary<SnippetProcessedData>, isPrivateSample?) {
     banner('Generating playlists');
 
-    /* Creating playlists directory */
-    status.add(`Creating \'${dir}\' folder`);
-    await rmRf(dir);
-    await mkDir(dir);
-    status.complete(true /*success*/, `Creating \'${dir}\' folder`);
+    if (!isPrivateSample) {
+        /* Creating playlists directory */
+        status.add(`Creating \'${dir}\' folder`);
+        await rmRf(dir);
+        await mkDir(dir);
+        status.complete(true /*success*/, `Creating \'${dir}\' folder`);
+    }
 
     const groups = groupBy(
```

Comment:
The lines below this is probably why the order is random. You are deleting item.order and modifying the group name of processedSnippets. Instead of deleting in the dictionary directly, can groupBy return a new dictionary?